src/p_saveg.c

```diff
@@ -141,7 +141,7 @@ static void saveg_write_pad(void)
 
     for (i=0; i<padding; ++i)
     {
-        saveg_read8();
+        saveg_write8(0);
```

Comment:
Oh, wow! Won't this break savegame compatibility?

Reply:
No, the main point is to move the pointer to a `padding` number of bytes. If it's a `0` entry or garbage, it doesn't matter.

Reply:
Only advances the `save_p` pointer, I get it. 👍